torchprime/experimental/torchax_models/deepseek_v3/model.py

```diff
@@ -590,71 +589,154 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
     return self.w2(F.silu(self.w1(x)) * self.w3(x))
 
 
-class MoE(nn.Module):
-  """
-  Mixture-of-Experts (MoE) module.
-
-  Attributes:
-      dim (int): Dimensionality of input features.
-      n_routed_experts (int): Total number of experts in the model.
-      n_local_experts (int): Number of experts handled locally in distributed systems.
-      n_activated_experts (int): Number of experts activated for each input.
-      gate (nn.Module): Gating mechanism to route inputs to experts.
-      experts (nn.ModuleList): List of expert modules.
-      shared_experts (nn.Module): Shared experts applied to all inputs.
-  """
-
-  def __init__(self, args: ModelArgs):
-    """
-    Initializes the MoE module.
-
-    Args:
-        args (ModelArgs): Model arguments containing MoE parameters.
-    """
+# class MoE(nn.Module):
```

Comment:
Should we keep this comment around or delete the old MoE class?

Reply:
done.